### PR TITLE
Remove test_req installations

### PR DESCRIPTION
**Pull request by ibrahimsharaf**
Issue (https://github.com/RaRe-Technologies/gensim/issues/1393)

### CODE REVIEW

**Comment by ibrahimsharaf** [comment]
@menshikh-iv should I add tensorflow to test_env in setup.py?

**Comment by menshikh-iv** [comment]
@ibrahimsharaf Yes, please add tensorflow to test_env

File: appveyor.yml [hunk 1]
```diff
@@ -66,8 +66,6 @@ test_script:
   # installed library.
   - "mkdir empty_folder"
   - "cd empty_folder"
-  - "pip install pyemd testfixtures unittest2 sklearn Morfessor==2.0.2a4"
```
Comment (menshikh-iv) [inline appveyor.yml]: Please revert this, because these dependencies will not be installed

**Comment by ibrahimsharaf** [comment]
@menshikh-iv do we have to specify the version? won't it get the latest one?

**Comment by menshikh-iv** [comment]
@ibrahimsharaf It's strange problem with installing dependencies, I will try to solve this.

**Comment by ibrahimsharaf** [comment]
@menshikh-iv I'll ping you if I reached a solution.

**Comment by ibrahimsharaf** [comment]
@menshikh-iv thoughts?

**Comment by menshikh-iv** [comment]
Thank you @ibrahimsharaf, now all ok.

**Pull request merged**